js/tsconfig.json

```diff
@@ -1,5 +0,0 @@
-{
-  "compilerOptions": {
-    "target": "esnext"
-  }
-}
```

Comment:
I’m not sure why the JS EngineBlock starter template would need a `tsconfig.json`. My understanding is that is doesn't.